docs/src/main/paradox/docs/delta/api/views/composite-sink.md

```diff
@@ -0,0 +1,163 @@
+# Composite Sinks
```

Comment:
Previously when thinking of the projection side of a composite view, we use the term "projection". Now there is much more overlap between "composite sink" and "projection" since the composite sink now regroups pretty much everything. There can also be the confusion with the actual target sink (ES, BG).